lib/Mailer/NotificationMailer.php

```diff
@@ -83,14 +83,19 @@ public function sendNotification(INotification $notification, $serverUrl, $email
 		$notificationObjectId = $notification->getObjectId();
 		$generatedId = "$notificationObjectType#$notificationObjectId";
 
+		$notificationLink = $notification->getLink();
+		if (empty($notificationLink)) {
```

Comment:
assuming the link will never be the string "0" this is ok.
still would be good to stop using `empty()` for strings as a habit

Reply:
Although I don't expect a link to be "0" (I think either it should start with "http" or "/"), I agree we should stop using `empty`. I'll be more strict from now on.